common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -90,6 +93,26 @@ def test_get_entitlements(self):
         results = response.data.get('results', [])
         assert results == CourseEntitlementSerializer(entitlements, many=True).data
 
+        # Make sure that these results aren't expired
+        assert results[0].get('expired_at') is None and results[1].get('expired_at') is None
+
+    def test_get_expired_entitlements(self):
+        past_datetime = datetime.utcnow().replace(tzinfo=pytz.UTC) - timedelta(days=365 * 2)
+        entitlements = CourseEntitlementFactory.create_batch(2, created=past_datetime)
+
+        # Set the first entitlement to be at a time that it isn't expired
+        entitlements[0].created = datetime.utcnow()
+        entitlements[0].save()
+
+        response = self.client.get(
+            self.entitlements_list_url,
+            content_type='application/json',
+        )
+        assert response.status_code == 200
+        results = response.data.get('results', [])
+        # Make sure that the first result isn't expired, and the second one is
+        assert results[0].get('expired_at') is None and results[1].get('expired_at')
```

Comment:
Consider using the same method used above to ensure the rest of the returned entitlement data is what you would expect.

` assert results == CourseEntitlementSerializer(course_entitlement).data`

Reply:
You can ignore this comment.